specification/digitaltwins/data-plane/Microsoft.DigitalTwins/stable/2023-10-31/digitaltwins.json

```diff
@@ -2305,6 +2308,13 @@
       "required": false,
       "type": "string"
     },
+    "timeoutInMinutes": {
+      "name": "timeoutInMinutes",
+      "in": "header",
+      "description": "Desired timeout for the delete job. If the timeout is reached the job with fail with an error, finish deletions currently in progress, and prevent any subsequent deletions from starting. Note this will leave your instance in an unknown state.",
```

Comment:
Should I mention a default time here or just leave it for our docs? left in out, in case we ever decided to change it

Reply:
another option for the description here:

"description": "Desired timeout for the delete job. If the timeout is reached the job with fail with an error, finish deleting any models, twins, or relationships that are currently in progress, and prevent any deletions from starting as part of this job. You may still send a new deletion job request later on. Note this will leave your instance in an unknown state."

Just want to make sure the behavior of the timeout is clear. Thoughts?